public/components/GeoPlot.vue

```diff
@@ -58,6 +58,15 @@
         C
       </a>
     </div>
+    <div
+      class="map-toggle"
+      :class="{ active: isSatelliteView }"
+      @click="mapToggle"
+    >
+      <a class="cluster-icon" title="Change Map" aria-label="Change Map">
```

Comment:
I would have made this a _button_ to be semantically correct, but I see the others are also _anchors_, so don't bother.